src/components/organisms/DeleteVendorPriceButton/DeleteVendorPriceButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useCallback } from 'react'
+import Button, { AppearanceTypes } from 'components/molecules/Button/Button'
+import { ReactComponent as Delete } from 'assets/icons/delete.svg'
+import { useTranslation } from 'react-i18next'
+import { includes, map } from 'lodash'
+import useAuth from 'hooks/useAuth'
+import { Privileges } from 'types/privileges'
+import { PriceObject } from '../forms/VendorPriceListForm/VendorPriceListForm'
+import { showNotification } from '../NotificationRoot/NotificationRoot'
+import { NotificationTypes } from 'components/molecules/Notification/Notification'
+import { showValidationErrorMessage } from 'api/errorHandler'
+import { useDeletePrices } from 'hooks/requests/useVendors'
+
+import classes from './classes.module.scss'
+
+export type DeleteVendorPriceButtonProps = {
+  languagePairModalContent: PriceObject[]
+  vendorId?: string
+}
+
+const DeleteVendorPriceButton: FC<DeleteVendorPriceButtonProps> = ({
+  languagePairModalContent = {},
+  vendorId,
+}) => {
+  const { t } = useTranslation()
+  const { userPrivileges } = useAuth()
+
+  const { deletePrices, isLoading: isDeletingPrices } =
+    useDeletePrices(vendorId)
+
+  const onDeletePrices = useCallback(async () => {
+    const payload = {
+      id: map(languagePairModalContent, ({ id }) => id),
```

Comment:
**[blocker]**

I think this will always delete all the skills under 1 language pair, but it should depend on which delete button is pressed. If the delete button next to the language pair is clicked, then it should delete all of them, but if the delete button on 1 of the skill rows is pressed, it should only remove that skill.